### PR TITLE
Set autoplace units to be the default on commit

### LINKED ISSUE #1169: First commit/deploy in admin model doesn't select 'automatically place'
In a real JES enabled model I deployed Wordpress, when to commit, and then deploy. I noticed however that it still had changes. It didn't auto-select "Automatically Place" as the default. This could be confusing to new users so we should make sure that it's the default.

### PR DESCRIPTION
**Pull request by hatched**
Having auto-place of units not be the default action was causing people to be confused as to why their services weren't starting. This sets auto-place as the default action, people in the know are still able to disable it if desired. Fixes #1169

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4106/
Test PASSed.

File: jujugui/static/gui/src/app/components/deployment/deployment.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ YUI.add('deployment-component', function() {
       // Setting a default state object.
       var state = {
         hasCommits: false,
-        autoPlace: !!localStorage.getItem('auto-place-default')
+        autoPlace: !localStorage.getItem('auto-place-default')
```
Comment (jaycee) [inline jujugui/static/gui/src/app/components/deployment/deployment.js]: Shouldn't this be "disable-auto-place" instead of "auto-place-default"?

**Comment by kadams54** [comment]
The QA steps as outlined work fine, however, setting the prefs has no effect, per @jaycee's comments in the code.

*QA Notes:* I worked through the steps outlined above, but then added some extra steps, which are necessary to confirm that setting prefs works properly:

1. Hit Shift + ? to bring up the help/prefs dialog.
2. Check "Default to not automatically place units on commit."
3. Click on the Save button.
4. Add a service to the canvas and check the deployment summary.
5. The Autodeploy option should not be selected this time.

**Comment by bac** [comment]
:+1:, no QA

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4112/
Test PASSed.

**Comment by jaycee** [comment]
:+1: with that change, thanks @hatched.

**Comment by kadams54** [comment]
QA is :ok:, :+1: on the code.

**Comment by hatched** [comment]
Thanks all! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**